### PR TITLE
Add @jonparrott as lead maintainer and @SethMichaelLarson as a maintainer

### PR DESCRIPTION
**Pull request by jonparrott**
Towards #1347

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -96,5 +98,6 @@ development <https://urllib3.readthedocs.io/en/latest/contributing.html#sponsors
 
 Sponsors include:
 
+- Google Cloud Platform (2018-present), sponsors `@jonparrott <https://github.com/jonparrott>`_'s work on an ongoing basis
 - Akamai (2017-present), sponsors `@haikuginger <https://github.com/haikuginger>`_'s work on an ongoing basis
```
Comment (shazow) [inline README.rst]: @haikuginger Let us know if we need to update this.

**Comment by SethMichaelLarson** [comment]
I'll see if I can get some sponsored time but I'm doubtful. :)

**Comment by jonparrott** [comment]
Oof, travis failed and I don't have the ability to merge without it.

**Comment by SethMichaelLarson** [comment]
Close-open to restart a build as a PR author. :)

**Comment by jonparrott** [comment]
I *am* a Travis admin so I can just restart the builds the failed there. :)

**Pull request merged**